internal/verifier/change_stream.go

```diff
@@ -189,10 +209,14 @@ func (verifier *Verifier) iterateChangeStream(ctx context.Context, cs *mongo.Cha
 }
 
 // StartChangeStream starts the change stream.
-func (verifier *Verifier) StartChangeStream(ctx context.Context) error {
+func (verifier *Verifier) StartChangeStream(ctx context.Context, batchSize *int32) error {
```

Comment:
Why does this need to be a setting? Aren’t we just taking whatever batch size we get from the source?

Reply:
I did this for a test to pass in a small batch size.

Reply:
Was that necessary, though?

`getMore` always returns after a relatively short period of inactivity with just whatever events it happens to have seen. So the batch size should be naturally limited to whichever events you’ve triggered … ?